io.sourceforge.chart-geany.chart-geany.yml

```diff
@@ -0,0 +1,54 @@
+app-id: io.sourceforge.chart_geany.chart_geany
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp # otherwise webengine is not available
+base-version: '5.15-21.08'
+command: ChartGeany
+desktop-file-name-suffix: '' # used to create development version
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=network
+  - --share=ipc
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
why do you "Flatpak" d-bus access.

Reply:
I removed it. Was copied from another project.